index.md

```diff
@@ -1,5 +1,7 @@
 # Mbed TLS documentation hub
 
+Mbed TLS provides an open-source implementation of cryptographic primitives, X.509 certificate handling and the SSL/TLS and DTLS protocols. It provides a reference implementation of the [PSA Cryptography API](https://arm-software.github.io/psa-api/crypto/). The project also supports the PSA Cryptoprocessor driver interface Specification which defines an interface for cryptoprocessor drivers. The small code footprint makes the project suitable for embedded systems. It has many users, including [TF-A](https://www.trustedfirmware.org/projects/tf-a/), [TF-M](https://www.trustedfirmware.org/projects/tf-m/) and [OP-TEE](https://www.trustedfirmware.org/projects/op-tee/).
```

Comment:
It's a bit odd to have `Specification` in title case with `driver interface` in lower case. Either `specification` or `Drive Interface`